model/model.go

```diff
@@ -654,11 +665,21 @@ type (
 		UpdatedAt    time.Time      `json:"updated_at"`
 	}
 
-	// InitiatePayoutRequest schema
-	InitiatePayoutRequest struct {
-		Currency string                  `json:"currency"`
-		Remarks  string                  `json:"remarks"`
-		Accounts []InitiatePayoutAccount `json:"accounts"`
+	// InitiateBulkPayoutRequest schema
+	InitiateBulkPayoutRequest struct {
+		Currency        string                       `json:"currency"`
+		Remarks         string                       `json:"remarks"`
+		Accounts        []BulkPayoutRecipientAccount `json:"accounts"`
+		BeneficiaryType PayoutType                   `json:"beneficiary_type"`
+		BeneficiaryID   *string                      `json:"beneficiary_id"`
```

Comment:
You don't need to add `omitempty` to this?

Reply:
I will add them accordingly